src/GitLab/Types.hs

```diff
@@ -443,22 +468,13 @@ instance FromJSON Iso8601Time where
     tryFormats alternateFormats <|> fail "could not parse ISO-8601 date"
     where
       tryFormat f =
-        case parseTime defaultTimeLocale f (T.unpack t) of
+        case parseTimeM True defaultTimeLocale f (T.unpack t) of
           Just d -> pure $ Iso8601Time d
           Nothing -> empty
       tryFormats = foldr1 (<|>) . map tryFormat
       alternateFormats = ["%FT%T%QZ", "%FT%T%Q%z"]
   parseJSON v = typeMismatch "UTCTime" v
 
-instance ToJSON Day where
-  toJSON = toJSON . formatTime defaultTimeLocale "%F"
-
-instance FromJSON Day where
-  parseJSON = withText "Day" $ \t ->
-    case parseTime defaultTimeLocale "%F" (T.unpack t) of
-      Just d -> pure d
-      Nothing -> empty
-
```

Comment:
They're now in `aeson`? We probably should set a lower bound for `aeon` for this change.